projects/public-search/src/app/search-bar/search-bar.component.ts

```diff
@@ -1,17 +1,27 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2020 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
 import { Component, Input, OnInit } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
-import { MainTitlePipe } from '../../../../admin/src/app/pipe/main-title.pipe';
-
-
-function escapeRegExp(data) {
-  return data.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&'); // $& means the whole matched string
-}
+import { SearchBarConfigService } from 'projects/shared/service/search-bar-config.service';
 
 @Component({
   selector: 'public-search-search-bar',
-  templateUrl: './search-bar.component.html',
-  styleUrls: ['./search-bar.component.css'],
```

Comment:
search-bar.component.css is empty so it could be removed.
I can't place a remark on an empty file

Reply:
+1